tests/bluetooth/iso/modules/iso_broadcast_src.c

```diff
@@ -0,0 +1,429 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdlib.h>
+#include <ctype.h>
+#include <getopt.h>
+#include <unistd.h>
+#include <zephyr/kernel.h>
```

Comment:
Nitpick, but I would prefer a blank line in between generic C and zephyr includes